doc/browsing_cubes/intro.rst

```diff
@@ -1,9 +1,54 @@
 Browsing Cubes
 ==============
 
-This is a description of how you can use the cube_browser package to look at cubes and make plots and sliders without having to make any widgets.
+The Cube Browser package offers an API which you can use to visualize your cubes in a Jupyter notebook.
+It utilizes the Python, Matplotlib, Iris and Cartopy libraries while offering the additional functionality of plotting an entire cube without having to slice it.
 
-Links to Notebooks
-------------------
+You can make use of the Cube Browser library to write a short notebook to display your Iris cube data.
+There are several advantages of using this approach, but the main advantage is flexibility in the options you can apply to each plot.
+Additionally, if you know what plots you would like and how you would like them laid out, this is a very quick way of producing your plots and exploring your cube data.
 
-four_axes.link
\ No newline at end of file
+Making Your Plots
+-----------------
+
+**Prepare your cube**
+
+Before you make your plot, you first have to choose and load the cubes you would like to visualize.  This is no different to the way you would do this in your normal work flow.
+If you need to perform any specific operations on the cube, such as aggregating, collapsing or cube maths, this should also be done before your plot.
+
+**Configure axes**
+
+Cube Browser uses matplotlib functionality, so you must set up your axes in the layout that you desire for your plots.
+The steps that are required here are that you define your projection
+(you can use the Iris convenience function `iplt.default_projection <http://scitools.org.uk/iris/docs/latest/iris/iris/plot.html#iris.plot.default_projection>`_ if this is appropriate),
```

Comment:
If projection is specifically referring to a "map" projection you could change it to "define your map projection" for clarity